src/main/java/com/emc/pravega/perf/PravegaTransactionWriterWorker.java

```diff
@@ -34,30 +34,33 @@
     PravegaTransactionWriterWorker(int sensorId, int eventsPerSec,
                                    int secondsToRun, boolean isRandomKey,
                                    int messageSize, Instant start,
-                                   ClientFactory factory, PerfStats stats,
-                                   String streamName,
-                                   long totalEvents,
+                                   PerfStats stats, String streamName,
+                                   long totalEvents, ClientFactory factory,
                                    int transactionsPerCommit) {
 
         super(sensorId, eventsPerSec, secondsToRun, isRandomKey,
-                messageSize, start, factory, stats, streamName, totalEvents);
+                messageSize, start, stats, streamName, totalEvents, factory);
 
         this.transactionsPerCommit = transactionsPerCommit;
         eventCount = new AtomicInteger(0);
```

Comment:
This is masking eventCount for the super class

Reply:
thanks a lot Tom, I have changed this to transEventCount